code/modules/mob/living/carbon/carbon_movement.dm

```diff
@@ -0,0 +1,45 @@
+/mob/living/carbon/movement_delay()
+	. = ..()
+	if(legcuffed)
+		. += legcuffed.slowdown
+
+	var/obj/item/organ/internal/cyberimp/chest/thrusters/J = getorganslot("thrusters")
+	if(istype(J) && J.on)
+		. -= 2
+
+
+var/const/NO_SLIP_WHEN_WALKING = 1
+var/const/SLIDE = 2
+var/const/GALOSHES_DONT_HELP = 4
+
+/mob/living/carbon/slip(s_amount, w_amount, obj/O, lube)
+	add_logs(src,, "slipped",, "on [O ? O.name : "floor"]")
+	return loc.handle_slip(src, s_amount, w_amount, O, lube)
+
+
+/mob/living/carbon/Process_Spacemove(movement_dir = 0)
+	if(..())
+		return 1
+	if(!isturf(loc))
+		return 0
+
+	// Do we have a jetpack implant (and is it on)?
+	var/obj/item/organ/internal/cyberimp/chest/thrusters/T = getorganslot("thrusters")
```

Comment:
RR's concerns have not been met. I do not think she would allow a merge until they are. Essentially it looks like you are duplicating the jetpack's checks (mode, istype, etc). You can simply add this portion of your code to the get_jetpack() proc as RR requested - there is not a need for a second istype() and other checks when you can consolidate them into fewer lines. This is rather important for procs which are called rapidly.